{
    "title" : "Unable to add source",
    "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "printableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "clickUri" : "https://na61.salesforce.com/kA132000000L8qD",
    "uniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "excerpt" : "When you try to add a source, the following error shows: ... Unable to add source 'YOURSOURCE\" to workgroup \"YOURWORKGROUP\" ... Environment ... Cause ... As can be seen on status.coveo.com , we mig...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1292,
    "percentScore" : 80.88122,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Unable to add source",
      "sysurihash" : "ocTgroNrsB0ð5Wi6",
      "urihash" : "ocTgroNrsB0ð5Wi6",
      "parents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Unable to add source\" uri=\"https://na61.salesforce.com/kA132000000L8qD\" /></parents>",
      "sfdcarticletype" : "Problem/Fix",
      "sfresolutionc" : "As a first step, check the administration console and administration API status <a href=\"http://status.coveo.com/\" target=\"_blank\">here</a>.<br><br>If both are operational, please contact coveo support.",
      "sysuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
      "sysprintableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
      "sflastpublisheddate" : 1488813990000,
      "systransactionid" : 161138,
      "sfurlname" : "2113",
      "sfsystemmodstamp" : 1513683920000,
      "sfisvisibleinapp" : "true",
      "sfcoveoversionc" : "Cloud Platform V2",
      "sysconcepts" : "administration ; coveo ; infrastructure provider ; error shows ; workgroup ; Unable ; Resolution ; Cause ; Environment",
      "sfkbversionnumber" : "1",
      "concepts" : "administration ; coveo ; infrastructure provider ; error shows ; workgroup ; Unable ; Resolution ; Cause ; Environment",
      "printableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
      "documenttype" : "article",
      "sfkbarticlekavid" : "ka132000000LMZkAAO",
      "sysindexeddate" : 1518187552000,
      "sfknowledgearticleid" : "kA132000000L8qDCAS",
      "sfcreatedbyid" : "00532000005ZlvdAAC",
      "sfconnectorc" : "N/A",
      "permanentid" : "bb6f91585bc4e59a1ac01c68d040f878d1a847df4d00022176ffbccfd737",
      "syslanguage" : [ "English" ],
      "transactionid" : 161138,
      "sfisvisibleinprm" : "true",
      "title" : "Unable to add source",
      "sfversionnumber" : 1,
      "sfarticlenumber" : "000002113",
      "date" : 1513683920000,
      "objecttype" : "KB Article",
      "allmetadatavalues" : "{\"kav_lastmodifiedby.attribute.type\": [\"User\"], \"coveo_short_id\": [\"kA132000000L8qD\"], \"sflastmodifiedbyname\": [\"Frederic Cote\"], \"kav_coveo_version__c__html_stripped\": [\"Cloud Platform V2\"], \"fileextension\": [\"\"], \"sfdcexpandedproduct\": [\"All;All|Cloud Platform\"], \"kav_createdby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00532000005ZlvdAAC\"], \"sflanguage\": [\"en_US\"], \"kav_createdby.attribute.type\": [\"User\"], \"kav_createdby.id\": [\"00532000005ZlvdAAC\"], \"kav_language\": [\"en_US\"], \"sfsitecoreversionc\": [\"N/A\"], \"sourcetype\": [\"Salesforce\"], \"title\": [\"Unable to add source\"], \"source\": [\"Salesforce Public Items\"], \"kav_owner.attribute.url\": [\"/services/data/v34.0/sobjects/User/00532000005ZlvdAAC\"], \"parents\": [\"<?xml version=\\\"1.0\\\" encoding=\\\"utf-16\\\"?><parents><parent name=\\\"Organization\\\" uri=\\\"https://na61.salesforce.com/home/home.jsp\\\" /><parent name=\\\"KB_Article__ka\\\" uri=\\\"http://www.salesforce.com/org:organization/articletype:KB_Article\\\" /><parent name=\\\"Unable to add source\\\" uri=\\\"https://na61.salesforce.com/kA132000000L8qD\\\" /></parents>\"], \"sfisvisibleinprm\": [true], \"kav_title\": [\"Unable to add source\"], \"kav_isvisibleincsp\": [true], \"hashtmlversion\": [true], \"kav_versionnumber\": [1], \"sfcreatedbyname\": [\"Pierre-Guillaume Laurin\"], \"sflastpublisheddate\": [1488813990], \"objecttypelabel\": [\"KB Article\"], \"kav_cause__c__html_stripped\": [\"As can be seen on status.coveo.com , we might be encountering an issue with our infrastructure provider\"], \"originalhtmlcharset\": [1200], \"coveo_connector_hasbinarydata\": [false], \"kav_lastpublisheddate\": [1488813990], \"sfsystemmodstamp\": [1513683920], \"ka_attribute.type\": [\"KB_Article__ka\"], \"sfdcproduct\": [\"Cloud Platform\"], \"kav_resolution__c__html_stripped\": [\"As a first step, check the administration console and administration API status  here . \\n \\n If both are operational, please contact coveo support.\"], \"clickableuri\": [\"https://na61.salesforce.com/kA132000000L8qD\"], \"sftitle\": [\"Unable to add source\"], \"kav_owner.id\": [\"00532000005ZlvdAAC\"], \"sfisvisibleinapp\": [true], \"ka_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__ka/kA132000000L8qDCAS\"], \"sfdcarticletype\": [\"Problem/Fix\"], \"kav_cause__c\": [\"As can be seen on status.coveo.com , we might be encountering an issue with our infrastructure provider\"], \"sfislatestversion\": [true], \"sfcausec\": [\"As can be seen on status.coveo.com , we might be encountering an issue with our infrastructure provider\"], \"contenttype\": [\"binarydata\"], \"sfurlname\": [\"2113\"], \"ka_id\": [\"kA132000000L8qDCAS\"], \"kav_datacategoryselections.datacategoryname\": [\"Configuration;Cloud_Platform\"], \"coveo_url\": [\"https://na61.salesforce.com/kA132000000L8qD\"], \"kav_urlname\": [\"2113\"], \"kav_dc_expanded_product\": [\"All;All|Cloud Platform\"], \"foldingcollection\": [\"29ad613626d27676db0d9d7878b12a692235a3cade6d43bab863e9101d32\"], \"body\": [\"<html>   <body>     <div style=\\\"background-color: #fff;border: 2px solid #BBC5CC;border-radius: 4px;float: left;font-family: lato;margin-left: 15px;margin-right: 15px;margin-bottom: 4%;padding: 10px;width: 60%;\\\">       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Description         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> When you try to add a source, the following error shows:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUnable to add source &#39;YOURSOURCE&quot; to workgroup &quot;YOURWORKGROUP&quot;</pre>\\n<br>\\u00a0</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Environment         </div>       </div>       <br/>       <table>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Product</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Sitecore Version</td>           <td>N/A</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Connector</td>           <td>N/A</td>         </tr>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Published</td>           <td>1488813990</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Version</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Build</td>           <td></td>         </tr>       </table>       <br/>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Cause         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> As can be seen on status.coveo.com , we might be encountering an issue with our infrastructure provider</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Resolution         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> As a first step, check the administration console and administration API status <a href=\\\"http://status.coveo.com/\\\" target=\\\"_blank\\\">here</a>.<br><br>If both are operational, please contact coveo support.</div>     </div>   </body> </html>\"], \"sfpublishstatus\": [\"Online\"], \"sfcreatedbyid\": [\"00532000005ZlvdAAC\"], \"kav_description__c__html_stripped\": [\"When you try to add a source, the following error shows: \\n \\u00a0\\n \\nUnable to add source 'YOURSOURCE\\\" to workgroup \\\"YOURWORKGROUP\\\" \\n \\n \\u00a0\"], \"kav_islatestversion\": [true], \"kav_knowledgearticleid\": [\"kA132000000L8qDCAS\"], \"sfknowledgearticleid\": [\"kA132000000L8qDCAS\"], \"sfownername\": [\"Pierre-Guillaume Laurin\"], \"kav_datacategoryselections.attribute.type\": [\"KB_Article__DataCategorySelection;KB_Article__DataCategorySelection\"], \"detectedtitle\": [\"Unable to add source 'YOURSOURCE\\\" to workgroup \\\"YOURWORKGROUP\\\"\"], \"kav_createddate\": [1488379006], \"language\": [\"English\"], \"coveo_foldingchild\": [\"1e7d572f37bd7d368d3fcd660b4162982d72b5dee118652d6540b17ba68a\"], \"sfownerid\": [\"00532000005ZlvdAAC\"], \"kav_createdby.coveo_record_size\": [2], \"kav_systemmodstamp\": [1513683920], \"objecttype\": [\"KB Article\"], \"connectortype\": [\"Salesforce2\"], \"kav_coveo_record_size\": [4], \"originaluri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US\"], \"sfisvisibleincsp\": [true], \"convertertype\": [\"TYPE_HTML\"], \"filetype\": [\"SalesforceItem\"], \"kav_owner.name\": [\"Pierre-Guillaume Laurin\"], \"sfcoveoversionc\": [\"Cloud Platform V2\"], \"sfdcexpandedarticletype\": [\"All;All|Problem/Fix\"], \"objecttypelabelplural\": [\"KB Articles\"], \"sfcurrencyisocode\": [\"CAD\"], \"sfisdeleted\": [false], \"extractedsize\": [1028], \"size\": [3020], \"kav_publishstatus\": [\"Online\"], \"kav_connector__c\": [\"N/A\"], \"generateexcerpt\": [true], \"sfdescriptionc\": [\"When you try to add a source, the following error shows:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUnable to add source &#39;YOURSOURCE&quot; to workgroup &quot;YOURWORKGROUP&quot;</pre>\\n<br>\\u00a0\"], \"sfarticletype\": [\"KB_Article__kav\"], \"kav_articletype\": [\"KB_Article__kav\"], \"kav_lastmodifieddate\": [1488813990], \"documenttype\": [\"article\"], \"sfkbarticlekavid\": [\"ka132000000LMZkAAO\"], \"kav_currencyisocode\": [\"CAD\"], \"foldingparent\": [\"1e7d572f37bd7d368d3fcd660b4162982d72b5dee118652d6540b17ba68a\"], \"mappingtype\": [\"KB_Article\"], \"kav_isvisibleinpkb\": [true], \"kav_sourceid\": [\"50032000019xOlSAAU\"], \"sfkbversionnumber\": [1], \"kav_isvisibleinapp\": [true], \"kav_resolution__c\": [\"As a first step, check the administration console and administration API status <a href=\\\"http://status.coveo.com/\\\" target=\\\"_blank\\\">here</a>.<br><br>If both are operational, please contact coveo support.\"], \"coveo_objecttype\": [\"KB_Article\"], \"printableuri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US\"], \"kav_datacategoryselections.attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000q30XAAQ;/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000q30WAAQ\"], \"kav_id\": [\"ka132000000LMZkAAO\"], \"sfresolutionc\": [\"As a first step, check the administration console and administration API status <a href=\\\"http://status.coveo.com/\\\" target=\\\"_blank\\\">here</a>.<br><br>If both are operational, please contact coveo support.\"], \"foldingchild\": [\"1e7d572f37bd7d368d3fcd660b4162982d72b5dee118652d6540b17ba68a\"], \"sfcreateddate\": [1488379006], \"kav_sitecore_version__c\": [\"N/A\"], \"kav_owner.coveo_record_size\": [2], \"kav_coveo_effective_modstamp\": [1513683920], \"kav_summary__html_stripped\": [\"\"], \"kav_dc_article_type\": [\"Problem/Fix\"], \"sfconnectorc\": [\"N/A\"], \"kav_dc_product\": [\"Cloud Platform\"], \"ka_coveo_record_size\": [2], \"kav_dc_expanded_article_type\": [\"All;All|Problem/Fix\"], \"kav_datacategoryselections.coveo_record_size\": [\"2;2\"], \"concepts\": [\"administration ; coveo ; infrastructure provider ; error shows ; workgroup ; Unable ; Resolution ; Cause ; Environment\"], \"sfkbid\": [\"kA132000000L8qDCAS\"], \"kav_firstpublisheddate\": [1488813990], \"coveo_organization_id\": [\"00D3000000007r2\"], \"coveo_foldingcollection\": [\"29ad613626d27676db0d9d7878b12a692235a3cade6d43bab863e9101d32\"], \"sffirstpublisheddate\": [1488813990], \"sflastmodifieddate\": [1488813990], \"kav_coveo_version__c\": [\"Cloud Platform V2\"], \"kav_vendor_version__c__html_stripped\": [\"\"], \"detectedlanguage\": [1], \"kav_isvisibleinprm\": [true], \"kav_owner.attribute.type\": [\"Name\"], \"kav_lastmodifiedby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sitename\": [\"Community\"], \"kav_createdby.name\": [\"Pierre-Guillaume Laurin\"], \"sfarticlenumber\": [\"000002113\"], \"kav_lastmodifiedby.coveo_record_size\": [2], \"sfisvisibleinpkb\": [true], \"kav_datacategoryselections.id\": [\"02o32000000q30XAAQ;02o32000000q30WAAQ\"], \"kav_description__c\": [\"When you try to add a source, the following error shows:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\nUnable to add source &#39;YOURSOURCE&quot; to workgroup &quot;YOURWORKGROUP&quot;</pre>\\n<br>\\u00a0\"], \"kav_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__kav/ka132000000LMZkAAO\"], \"kav_articlenumber__stripped\": [\"2113\"], \"sfversionnumber\": [1], \"kav_attribute.type\": [\"KB_Article__kav\"], \"kav_validationstatus\": [\"Reviewed\"], \"coveo_foldingparent\": [\"1e7d572f37bd7d368d3fcd660b4162982d72b5dee118652d6540b17ba68a\"], \"collection\": [\"default\"], \"sfsourceid\": [\"50032000019xOlSAAU\"], \"date\": [1513683920], \"kav_lastmodifiedby.id\": [\"00560000003q7JgAAI\"], \"kav_datacategoryselections.datacategorygroupname\": [\"Article_Type;Product\"], \"conversionstate\": [0], \"sfvalidationstatus\": [\"Reviewed\"], \"kav_isdeleted\": [false], \"sflastmodifiedbyid\": [\"00560000003q7JgAAI\"], \"kav_lastmodifiedby.name\": [\"Frederic Cote\"], \"kav_articlenumber\": [\"000002113\"], \"permanentid\": [\"bb6f91585bc4e59a1ac01c68d040f878d1a847df4d00022176ffbccfd737\"]}",
      "sflastmodifiedbyid" : "00560000003q7JgAAI",
      "syssfownerid" : "00532000005ZlvdAAC",
      "sfsitecoreversionc" : "N/A",
      "sourcetype" : "Salesforce",
      "sysconnectortype" : "Salesforce2",
      "sftitle" : "Unable to add source",
      "rowid" : 5492598,
      "syssfcreatedbyid" : "00532000005ZlvdAAC",
      "sfdescriptionc" : "When you try to add a source, the following error shows:<br> \n<pre class=\"ckeditor_codeblock\">\nUnable to add source &#39;YOURSOURCE&quot; to workgroup &quot;YOURWORKGROUP&quot;</pre>\n<br> ",
      "size" : 3020,
      "sfislatestversion" : "true",
      "sysdocumenttype" : "article",
      "sfcreateddate" : 1488379006000,
      "sfsourceid" : "50032000019xOlSAAU",
      "clickableuri" : "https://na61.salesforce.com/kA132000000L8qD",
      "syssource" : "Salesforce Public Items",
      "sfownerid" : "00532000005ZlvdAAC",
      "orderingid" : 212384990660371405,
      "sfpublishstatus" : "Online",
      "syssize" : 3020,
      "sysdate" : 1513683920000,
      "sflanguage" : "en_US",
      "sysparents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Unable to add source\" uri=\"https://na61.salesforce.com/kA132000000L8qD\" /></parents>",
      "sfdcexpandedarticletype" : "All;All|Problem/Fix",
      "sfcurrencyisocode" : "CAD",
      "sfkbid" : "kA132000000L8qDCAS",
      "sfcausec" : "As can be seen on status.coveo.com , we might be encountering an issue with our infrastructure provider",
      "sfdcexpandedproduct" : "All;All|Cloud Platform",
      "syssfcreateddate" : 1488379006000,
      "sflastmodifieddate" : 1488813990000,
      "sfisvisibleinpkb" : "true",
      "source" : "Salesforce Public Items",
      "sfisdeleted" : "false",
      "sfisvisibleincsp" : "true",
      "sfvalidationstatus" : "Reviewed",
      "sflastmodifiedbyname" : "Frederic Cote",
      "collection" : "default",
      "sfdcproduct" : [ "Cloud Platform" ],
      "syssourcetype" : "Salesforce",
      "sfarticletype" : "KB_Article__kav",
      "indexeddate" : 1518187552000,
      "connectortype" : "Salesforce2",
      "filetype" : "SalesforceItem",
      "sfcreatedbyname" : "Pierre-Guillaume Laurin",
      "sysclickableuri" : "https://na61.salesforce.com/kA132000000L8qD",
      "sysfiletype" : "SalesforceItem",
      "language" : [ "English" ],
      "sffirstpublisheddate" : 1488813990000,
      "sfownername" : "Pierre-Guillaume Laurin",
      "sitename" : "Community",
      "sysrowid" : 5492598,
      "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
      "syscollection" : "default"
    },
    "Title" : "Unable to add source",
    "Uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "PrintableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "ClickUri" : "https://na61.salesforce.com/kA132000000L8qD",
    "UniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA132000000L8qDCAS/language:en_US",
    "Excerpt" : "When you try to add a source, the following error shows: ... Unable to add source 'YOURSOURCE\" to workgroup \"YOURWORKGROUP\" ... Environment ... Cause ... As can be seen on status.coveo.com , we mig...",
    "FirstSentences" : null
  }